frontstage/views/secure_messaging/messages_get.py

```diff
@@ -17,12 +17,14 @@
 @secure_message_bp.route('/messages/', methods=['GET'])
 @secure_message_bp.route('/messages/<label>', methods=['GET'])
 @jwt_authorization(request)
-def messages_get(session, label="INBOX"):
+def messages_get(session, label="INBOX", new_message=False):
     messages_list = get_messages_list(label)
     messages = messages_list['messages']
+    if request.args.get('new_message'):
```

Comment:
```Python
try:
    new_message = request.args['new_message']
except KeyError:
	sys.exc_clear() # clears the exception - we don't need it

return render_template('secure-messages/secure-messages.html',
					   _theme='default',
                       messages=messages['messages'],
					   links=messages['_links'],
					   label=label,
                       total=unread_msg_total,
					   new_message=new_message)

Reply:
I don't think the exception is needed, `new_message = request.args.get('new_message', None)` should work.